cmd_worlds.lua

```diff
@@ -0,0 +1,26 @@
+function HandleWorldsCommand(Split, Player)
+	local NumWorlds = 0
+	local Worlds = {}
+	local Response = {}
+	
+	local ListWorld = function(World)
+		NumWorlds = NumWorlds + 1
+		table.insert(Worlds, World:GetName())
```

Comment:
You already have an index to the table available, use that instead of `table.insert`.